### ISSUE TITLE
Should benchmark_grid allow instantiated resamplings if they are compatible with all tasks?

### ISSUE THREAD

**Issue by mb706** [mlr-org/mlr3#392]
```r
> benchmark_grid(tsk("iris"), lrn("classif.rpart"), rsmp("cv")$instantiate(tsk("iris")))
Error: Resampling 'cv' may not be instantiated
```
what if I want to compare different learners on tasks that all have the same number of rows / blocking pattern etc?

**Comment by mllg** [comment]
This is a very rare case, and you can always construct the grid yourself if required. If you happen to need this frequently, re-open.

**Comment by mllg** [comment]
I might have been wrong, just needed the exact same thing. Implemented in 644afda.
